joy2twist/src/joy2twist_node.cpp

```diff
@@ -0,0 +1,86 @@
+#include "joy2twist/joy2twist_node.hpp"
+
+namespace joy2twist
+{
+Joy2TwistNode::Joy2TwistNode(std::shared_ptr<ros::NodeHandle> nh) : nh_(std::move(nh))
+{
+  load_parameters();
+
+  joy_sub_ = nh_->subscribe("/joy", 1, &Joy2TwistNode::joy_cb, this);
+  twist_pub_ = nh_->advertise<MsgTwist>("/cmd_vel", 1);
```

Comment:
Also here I don't think that global names should be used.

Reply:
While using a private name for this topic `/joy2twist_node/cmd_vel`, we'd need to relaunch the driver and remap the cmd_vel topic there, aren't we?

Reply:
Topics remained being handled by the nh, while params are handled by the private_nh